kotlin-sdk/build.gradle.kts

```diff
@@ -10,24 +10,65 @@ plugins {
 
 val releaseVersion = project.extra["version"].toString()
 
+group = project.extra["groupId"].toString()
+version = releaseVersion
+
+kotlin {
+    androidTarget {
+        compilations.all {
+            compileTaskProvider.configure {
+                compilerOptions {
+                    jvmTarget.set(JvmTarget.JVM_11)
```

Comment:
Might want to bump that to JVM_17 (https://developer.android.com/build/jdks), unless the compatibility is really needed.

Reply:
@realdadfish setting this to JVM 17 would prevent JVM 11 users from adopting this library, wouldn't it? If so, I'd keep the target for this lib low to maximize compatibility and to not break anyone's toolchain setup.

[Temurin will support JDK 11 until 31 Oct 2027](https://endoflife.date/eclipse-temurin), so for now it's far from being dead.

Reply:
I would stick to `JVM_11`, see my [comment](https://github.com/open-feature/kotlin-sdk/pull/148#issuecomment-2931141941)